R/visr.R

```diff
@@ -301,7 +301,9 @@ visr.survfit <- function(
 
   yscaleFUN <- function(x) sprintf("%.2f", x)
 
-  gg <- ggplot2::ggplot(tidy_object, ggplot2::aes(x = time, group = strata)) +
+  gg <- ggplot2::ggplot(tidy_object, ggplot2::aes(x = time, 
```

Comment:
why add the fill = strata argument?

Reply:
If this is not specified at a `ggplot2::ggplot()` level, I couldn't find any other way to adjust the alpha values after plotting. Now I can extract the current colours, including their alphas, and apply the modified values as a `scale_colour_manual()`/`scale_fill_manual()`. This way, I only modify the actual ggplot object without creating a new one from scratch afterwards.